recipes/nsis/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "2.46" %}
+
+package:
+  name: nsis
+  version: {{ version }}
+
+source:
+  fn: nsis-{{ version }}.zip
+  url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
+  sha256: ced6561f8aed81c8f3d6bc5a33684e03ca36a618110c0a849880c703337f26cc
+
+build:
+  number: 0
+  skip: true  # [unix]
+
+about:
+  home: http://sourceforge.net/projects/nsis
+  summary: Nullsoft Scriptable Install System
+  license: CPL-1.0, zlib, bzip2
```

Comment:
May want to include the [license]( https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/branches/nsis2/COPYING ) file for good measure. Then again it may already be in this source.